src/Keboola/StorageApi/Client.php

```diff
@@ -2112,4 +2113,12 @@ public function isAwsDebug()
     {
         return $this->awsDebug;
     }
+
+    private function verifyIsPositiveWholeNumber($value, $errorMessage)
+    {
+        if (((string) (int) $value === (string) $value) && ($value > 0)) {
```

Comment:
Prosím tě zkontroluj mi znamínko :grimacing: jinak si myslím, že tohle je totálně bulletproof

Reply:
asi je, ale stejne me to prijde jako blbost :)

Zavadi to uplne zbytecnej predpoklad, ze file id je cislo, cimz si uplne zbytecne naserem do gati pokud z nej nekdy budem chtet udelat neco jinyho. Kdyz das kteroukoliv z tech veci krome `null` do api callu https://keboola.docs.apiary.io/#reference/files/manage-files/file-detail tak to uplne pohodove vrati "File not found" (i na hodnotu `-243 karlovych hoven`). Jestli file existuje stejne nejses v klientovi schopnej osetrit. Takze jediny o co jde z pohledu klienta je, zaridit, aby ta hodnota nebyla prazdna, aby se nestalo, ze klient zavola uplne jinej API call a vrati uplne nesmyslnou odpoved. 

For everything else, there’s API. https://www.youtube.com/watch?v=0v7D_SirqTc

Reply:
Je to tak:)
![image](https://user-images.githubusercontent.com/903531/63505830-8bdae480-c4d4-11e9-923e-bba987308f4c.png)

Reply:
Tohle teda projde, ale je to věc backendu. 
![image](https://user-images.githubusercontent.com/903531/63505919-b7f66580-c4d4-11e9-9b0b-a3911f3a6b12.png)

Reply:
Dal jsem tam `empty($fileId)` ...